.github/workflows/github-actions-release-candidate.yml

```diff
@@ -146,4 +146,4 @@ jobs:
               });
             } else {
               console.log('No pull request found for this commit');
-            }
+            }
```

Comment:
Can you run `git checkout origin/master -- .github/workflows/github-actions-release-candidate.yml` and push that up to get rid of this change.